app/assets/json/translations/react/en.json

```diff
@@ -60,6 +64,12 @@
   "listings.prioritiesDescriptor.mobility": "Mobility Impairments",
   "listings.prioritiesDescriptor.mobilityHearingVision": "Mobility, Hearing and/or Vision Impairments",
   "listings.prioritiesDescriptor.vision": "Vision Impairments",
+  "listings.realtor_commission": "For the Buyer's Realtor",
+  "listings.realtor_commission_header": "",
+  "listings.realtor_commission_how_to": "",
+  "listings.realtor_commission_not_eligible": "",
```

Comment:
would rather leave these out for now than add them with empty values

Reply:
how do you feel about commenting them out? I'm thinking we'll need them soon-ish, since they're a part of additional info in sales listings, and leaving a `TODO` could help to make sure we don't forget to add these when the time comes

Reply:
nvm, json doesn't do comments 🤷